docs/usage/models.md

```diff
@@ -1162,21 +1162,24 @@ To do this, you may want to use a `default_factory`.
 Here is an example:
 
 ```py
-from datetime import datetime
+from datetime import datetime, timezone
 from uuid import UUID, uuid4
 
 from pydantic import BaseModel, Field
 
 
+def datetime_now() -> datetime:
+    return datetime.now(timezone.utc)
+
+
 class Model(BaseModel):
     uid: UUID = Field(default_factory=uuid4)
-    updated: datetime = Field(default_factory=datetime.utcnow)
+    updated: datetime = Field(default_factory=datetime_now)
 
 
 m1 = Model()
 m2 = Model()
 assert m1.uid != m2.uid
-assert m1.updated != m2.updated
```

Comment:
I don't know how this was passing before.

Reply:
Well, it kinda should produce different time each time the default factory is being called.

The following passes

```py
>>> from datetime import datetime
>>> assert datetime.utcnow() != datetime.utcnow()
```

Reply:
We have a mock on `datetime.now` on the `test_docs.py`, that's why it doesn't pass. 😅